esp/services/ws_rrd/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+
+
+# Component: ws_rrd
+#####################################################
+# Description:
+# ------------
+#    Cmake Input File for ws_rrd
+#####################################################
+
+project( ws_rrd )
+
+include(${HPCC_GM_SOURCE_DIR}/esp/scm/additional.cmake)
+
+set (    SRCS
+         ${HPCC_GM_SOURCE_DIR}/esp/scm/ws_rrd.ecm
+         ${ESPSCM_GENERATED_DIR}/ws_rrd_esp.cpp
+         ./WsRRDService.cpp
+         ./WsRRDPlugin.cpp
+         ./RRDFileHelper.cpp
+         ./RRDGraphWrapper.cpp
+    )
+
+include_directories (
+         ${HPCC_SOURCE_DIR}/esp/esplib
+         ${HPCC_SOURCE_DIR}/esp/platform
+         ${HPCC_SOURCE_DIR}/system/jlib
+         ${HPCC_SOURCE_DIR}/common/environment
+         ${HPCC_SOURCE_DIR}/esp/services
+         ${HPCC_SOURCE_DIR}/common
+         ${HPCC_SOURCE_DIR}/system/security/securesocket
+         ${HPCC_SOURCE_DIR}/system/security/shared
+         ${HPCC_SOURCE_DIR}/system/include
+         ${HPCC_SOURCE_DIR}/common/remote
+         ${HPCC_SOURCE_DIR}/clients
```

Comment:
The platform is a dependecy for building.  This project is built similiar to LN, with platform side by side.